docs/src/pages/[platform]/components/link/index.page.mdx

```diff
@@ -1,7 +1,7 @@
 ---
 title: Link
 description: Link renders an anchor element <a> by default and is primarily used for navigation.
-ariaPattern: https://www.w3.org/WAI/ARIA/apg/patterns/link
+ariaPattern: https://www.w3.org/WAI/ARIA/apg/patterns/link/
```

Comment:
What's the difference between adding a `/` and not adding one?

Reply:
without `/`, it returns `301` and redirects to the one with `/`